models/frameSkipResult.js

```diff
@@ -0,0 +1,15 @@
+const BaseModel = require('./baseModel.js');
+const Sequelize = require('sequelize');
+
+module.exports = class FrameSkipResult extends BaseModel {
+    /**
+     * Initialize this model
+     */
+    static init(sequelize) {
+        return super.init({
```

Comment:
I wonder if it would be better to have the curly on the next line to create more symmetry so it's clearer that { sequelize } is also an argument.

Reply:
fixed